raft/quorum/testdata/joint_vote.txt

```diff
@@ -0,0 +1,163 @@
+# Empty joint config wins all votes. This isn't used in production.
+vote cfgj=zero
```

Comment:
Is there a semantic difference in these tests between not including `cfg=` for the first majority quorum and setting `cfgj=zero` for the second?

Reply:
Yes, zero is a marker value precisely to tell the test harness to use a joint quorum with an empty half. I added comments at the top of both tests to alert the reader to that fact.